Meta/Lagom/Fuzzers/FuzzASN1.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2021, Brian Gianforcaro <bgianf@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCrypto/ASN1/ASN1.h>
+#include <LibCrypto/ASN1/PEM.h>
+#include <LibTLS/Certificate.h>
+#include <stddef.h>
+#include <stdint.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size)
+{
+    ReadonlyBytes buffer = { data, size };
+    auto decoded_certificate = Crypto::decode_pem(buffer);
```

Comment:
Why go through PEM? That'll just make it harder to tell where the issue actually is
If you want to fuzz the PEM decoder, I suggest making another fuzzer